python/cudf/cudf/core/column/lists.py

```diff
@@ -288,6 +289,23 @@ def _transform_leaves(self, func, *args, **kwargs) -> Self:
             )
         return lc
 
+    def to_pandas(
+        self,
+        *,
+        index: Optional[pd.Index] = None,
+        nullable: bool = False,
+    ) -> pd.Series:
+        # Can't rely on Column.to_pandas implementation for lists.
+        # Need to perform `to_pylist` to preserve list types.
```

Comment:
Presumably this is some bug in how list dtypes are handled in arrow? Do we need to report something upstream?

Reply:
There is already an open issue: https://github.com/apache/arrow/issues/34574